### ISSUE TITLE
Error: "The plugin needs write access [...]"

### ISSUE THREAD

**Issue by Shnalf** [Sloeber/arduino-eclipse-plugin#949]
![bildschirmfoto vom 2018-03-31 22-16-57](https://user-images.githubusercontent.com/37964113/38167189-882f288e-3531-11e8-9b13-fe9317c981f4.png)
When I start Eclipse I get an error message, that sais: "The plugin Needs write access to /opt/eclipse/arduinoPlugin". I've installed the V3 plugin on the Eclipse IDE for C/C++ Developers Version Neon.3 Release (4.6.3) on Linux Mint (Ubuntu).
Does anyone know, how I can fix it?

Many thanks
Ralf Rumbler

**Comment by jantje** [comment]
Did you try?
`chmod +w opt/eclipse/arduinoPlugin`

**Comment by Shnalf** [comment]
![4](https://user-images.githubusercontent.com/37964113/38171691-1f35b438-359f-11e8-806f-30b744cf202c.png)
I fixed it, by changing the access rights through the filesystem(?) of Linux Mint, by clicking rightclick on the folder, clicking on properties, going to "access rights" and selecting "create & delete files" at the folder access boxes (I've translated it from the german Linux Mint, so it could be a bit different in the english OS). Running `chmod +w` commands didn't do anything. I had to reinstall the plugin, because the installation couldn't be finished because of the missing access rights.

Thank you for your tip!
Ralf Rumbler
